mz_zip_rw.c

```diff
@@ -682,8 +682,11 @@ int32_t mz_zip_reader_entry_save_file(void *handle, const char *path) {
     /* Check if file exists and ask if we want to overwrite */
     if (reader->overwrite_cb && mz_os_file_exists(pathwfs) == MZ_OK) {
         err_cb = reader->overwrite_cb(handle, reader->overwrite_userdata, reader->file_info, pathwfs);
-        if (err_cb != MZ_OK)
+        if (err_cb != MZ_OK) {
+            if (err_cb == MZ_EXIST_ERROR)
+                err_cb = MZ_OK;
```

Comment:
Why do you need to set `err_cb = MZ_OK`?

Reply:
`err_cb` needs to be `MZ_OK` if we want to continue processing the zip file because `mz_zip_reader_save_all` exits immediately if anything other than MZ_OK is returned from `mz_zip_reader_entry_save_file`.

I thought about modifying the test in `mz_zip_reader_save_all` instead, but then it wouldn't be able to tell the difference between overwrite_cb returning skip and a `MZ_EXIST_ERROR` caused by something else in `mz_zip_reader_entry_save_file`.